### DISCUSSION TITLE
Improving error messages

### DISCUSSION THREAD

**Discussion by mauritskorse** [livewire/flux#360] [Feature Requests]
Right now I sometimes get an error message 'Could not find panel...'.
It would be great if these could be more informative by adding the panel name that was not found.
i.e.
```diff
    let panelEl = container.getPanel(panel);
-   if (!panelEl) throw new Error("Could not find panel...");
+   if (!panelEl) throw new Error("Could not find panel "+ panel);
```

**Comment by jtomlinson** [comment]
I ran into an issue today where having a more detailed error message like this would have been very helpful.
